src/js/components/ComponentList.js

```diff
@@ -45,9 +45,10 @@ class ComponentList extends React.Component {
   }
 
   getVisibleComponents(units, displayCount) {
-    let sortFunction = UnitHealthUtil.getHealthSortFunction;
-
-    units = units.sort(sortFunction('health'));
+    // HealthTypes gives the sorting weight.
+    units = units.sort(function (a, b) {
+      return HealthTypes[a.getHealth().title.toUpperCase()] - HealthTypes[b.getHealth().title.toUpperCase()];
+    });
```

Comment:
the efficiency god sheds a single tear (but this is unavoidable without loads of other changes so 🚢 )